### DISCUSSION TITLE
Your account is BANNED DUINO-COIN

### DISCUSSION THREAD

**Discussion by Giovanni-Cerra** [revoxhere/duino-coin#1520] [General]
Your account is BANNED.
You have violated our [ToS](https://github.com/revoxhere/duino-coin#terms-of-service)
If you think this is a mistake, please contact us at [duino.coin@gmail.com](mailto:duino.coin@gmail.com)

**Comment by Grantrocks** [comment]
What did you do? They would only ban you if you did something wrong. If you think this is TRULY a mistake i would recommended joining their discord and sending a dm or asking for help.
